src/MagnumPlugins/UfbxImporter/UfbxImporter.h

```diff
@@ -0,0 +1,354 @@
+@endcode
+
+See @ref building-plugins, @ref cmake-plugins, @ref plugins and
+@ref file-formats for more information.
+
+@section Trade-UfbxImporter-behavior Behavior and limitations
+
+The plugin supports @ref ImporterFeature::OpenData, @relativeref{ImporterFeature,OpenState} and @relativeref{ImporterFeature,FileCallback} features.
+Immediate dependencies are loaded during the initial import meaning the
+callback is called with @ref InputFileCallbackPolicy::LoadTemporary.
+In case of images, the files are loaded on-demand inside @ref image2D() calls
+with @ref InputFileCallbackPolicy::LoadTemporary and
+@ref InputFileCallbackPolicy::Close is emitted right after the file is fully
+read.
+
+The importer recognizes @ref ImporterFlag::Verbose if built in debug mode
+(@ref CORRADE_IS_DEBUG_BUILD or @cpp NDEBUG @ce if not defined). The verbose
+logging prints detailed ufbx-internal callstacks on load failure that can be
+used for debugging or reporting issues.
+
+@subsection Trade-UfbxImporter-behavior-scene Scene import
+
+-   ufbx supports only a single scene, though in practice it is extremely rare
+    to have FBX files containing more than a single scene.
+-   FBX files contain an implicit root node that is ignored unless the
+    @cb{.ini} preserveRootNode @ce @ref Trade-UfbxImporter-configuration "configuration option" is enabled.
+-   FBX files may contain nodes with "geometric transforms" that transform only
+    the mesh of the node without affecting children. These are converted to
+    unnamed helper nodes in the scene unless the @cb{.ini} geometricTransformNodes @ce
+    @ref Trade-UfbxImporter-configuration "configuration option" is explicitly disabled.
+-   Imported scenes always have @ref SceneMappingType::UnsignedInt, with
+    @ref SceneData::mappingBound() equal to @ref objectCount(). The scene is
+    always 3D.
+-   All reported objects have a @ref SceneField::Parent (of type
+    @ref SceneFieldType::Int), @ref SceneField::Translation (of type @ref SceneFieldType::Vector3d),
+    @ref SceneField::Rotation (of type @ref SceneFieldType::Quaterniond),
+    @ref SceneField::Scaling (of type @ref SceneFieldType::Vector3d) and
+    importer-specific @cpp "Visibility" @ce (of type @ref SceneFieldType::UnsignedByte
+    representing a boolean value).
+    These five fields share the same object mapping with
+    @ref SceneFieldFlag::ImplicitMapping set.
+-   If the scene references meshes, a @ref SceneField::Mesh (of type
+    @ref SceneFieldType::UnsignedInt) and a @ref SceneField::MeshMaterial (of
+    type @ref SceneFieldType::Int) is present, both with
+    @ref SceneFieldFlag::OrderedMapping set. Missing material IDs are @cpp -1 @ce.
+    If a mesh contains multiple materials it is split into parts and the node
+    contains each part as a separte mesh/material entry.
+    The same mesh can appear instanced multiple times under many
+    nodes with different materials. See @ref Trade-UfbxImporter-instancing for further details.
```

Comment:
I can't see the referenced section anywhere :)

Reply:
Welp, I somehow completely forgot to write it. Though if instancing meshes is well supported and expected within Magnum maybe it doesn't need it's own paragraph.

Reply:
Yeah, it probably doesn't, especially given you're going to remove `perInstanceMaterials` anyway. So just remove the three references to it.